src/btree/bt_io.c

```diff
@@ -365,6 +366,12 @@ __wt_bt_write(WT_SESSION_IMPL *session, WT_ITEM *buf,
 	WT_STAT_FAST_DATA_INCR(session, cache_write);
 	WT_STAT_FAST_CONN_INCRV(session, cache_bytes_write, dsk->mem_size);
 	WT_STAT_FAST_DATA_INCRV(session, cache_bytes_write, dsk->mem_size);
+	if (checkpoint_io) {
```

Comment:
@keithbostic on closer inspection, I don't think ``write_size`` is what I was looking for. I've added more detail here: https://jira.mongodb.org/browse/WT-2746?focusedCommentId=1316325&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-1316325